eldritch/mythos.py

```diff
@@ -412,6 +412,155 @@ def get_trigger(self, event, state):
     return None
 
 
+class Mythos16(Environment):
+  def __init__(self):
+    super().__init__(
+        "Mythos16", "WitchHouse", "Cave",
+        {"square", "diamond"}, {"circle"},
+        "urban", activity_location="Uptown"
+    )
+
+  def get_trigger(self, event, state):
+    if not isinstance(event, events.Movement):
+      return None
+    if event.character.place.name != self.activity_location:
+      return None
+    return events.Purchase(event.character, "unique", draw_count=2, keep_count=2)
```

Comment:
Technically, the card says that they _may_ draw 2 unique items and purchase some. I think the only time this might come into play is if there is something on the bottom of the deck that the Drifter wanted to scrounge up?